clients/config-center-client/src/main/java/org/apache/servicecomb/config/center/client/AddressManager.java

```diff
@@ -17,25 +17,32 @@
 
 package org.apache.servicecomb.config.center.client;
 
-import java.util.ArrayList;
 import java.util.List;
+import java.util.stream.Collectors;
 
 import org.apache.commons.lang.StringUtils;
+import org.apache.servicecomb.http.client.common.EndpointAddress;
 import org.apache.servicecomb.http.client.common.HttpUtils;
+import org.apache.servicecomb.http.client.event.CommonEventManager;
+import org.apache.servicecomb.http.client.event.RefreshEndpointEvent;
+
+import com.google.common.eventbus.Subscribe;
 
 public class AddressManager {
+
   public static final String DEFAULT_PROJECT = "default";
 
   private final String projectName;
 
-  private final List<String> addresses;
+  private EndpointAddress endpointAddress;
 
-  private int index = 0;
+  private boolean isSSLEnable = false;
 
   public AddressManager(String projectName, List<String> addresses) {
     this.projectName = StringUtils.isEmpty(projectName) ? DEFAULT_PROJECT : projectName;
-    this.addresses = new ArrayList<>(addresses.size());
-    addresses.forEach((address -> this.addresses.add(formatAddress(address))));
+    this.endpointAddress = new EndpointAddress(
+        addresses.stream().map(this::formatAddress).collect(Collectors.toList()));
+    CommonEventManager.register(this);
```

Comment:
Maybe add `public AddressManager(String projectName, List<String> addresses, EventBus eventBus)' is better.